index.html

```diff
@@ -262,6 +260,187 @@ <h2 class="contact-me-text">Contact me</h2>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div id="popupCard3" class="popup3 popup-div">
+        <div class="inner-popup">
+            <span class="close3 popout">&times;</span>
+            <h2>Multi-Post Stories</h2>
+            <div>
+                <ul class="canopy-info">
+                    <li class="canopy-text">CANOPY</li>
+                    <img src="img/Counter.svg" />
+                    <li>Back End Dev</li>
+                    <img src="img/Counter.svg" />
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="popup-img">
+                <img src="img/Snapshoot Portfolio popup3.svg" />
+            </div>
+            <div class="desktop-grid">
+                <p class="popup-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+                    book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+
+                    <p class="popup-text display-none">
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+                        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.
+                    </p>
+                <div class="tags-and-buttons-container">
+                    <ul class="languages-list lang-popup">
+                        <li class="html">Html</li>
+                        <li class="css">Css</li>
+                        <li class="javascript">Javascript</li>
+                    </ul>
+                    <ul class="language-list lang-popup plus-langs display-none">
+                        <li class="javascript one">ruby</li>
+                        <li class="javascript two">Bootstrap</li>
+                    </ul>
+
+                    <hr class="popup-hr">
+                    <div class="button">
+                        <button type="button" class="see-button seeLive">See live  <img src="icons/Icon.svg"/></button>
+                        <button type="button" class="see-button seeSource">See source <img src="icons/Vector.svg"/></button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] Just as i mentioned in the comment before, per the project requirements, the `modal popup` is to be populated with the data from the projects array after you are done generating the project section dynamically with data from the data array 💯 
- So kindly remove all of this content from the `index.html` file 👍